### ISSUE TITLE
Building iptable_mangle and iptable_raw

### ISSUE THREAD

**Issue by fl4shback** [mammo0/qnap-ip6tables_nat-module#11]
Hi! 

Sorry for raising an issue but I didn't find any other way to contact you.
[I cloned your repo](https://github.com/fl4shback/qnap-iptables-modules) to try to build mangle and raw modules but I can't seem to manage to build the modules.

Do you think it's possible to build those modules for QTS ?

Thank you !

**Comment by mammo0** [comment]
Hey @fl4shback!

normally it should be possible to build any standard modules for the QTS kernel.
What problem do you have?

**Comment by mammo0** [comment]
Ok, nice to hear that 👍
But why do you need my patch file? It only enables the IPv6 modules. But you wanted the raw and mangle modules?

Or do I miss something?

**Comment by fl4shback** [comment]
https://github.com/fl4shback/qnap-iptables-modules/commit/274d9dc3899ff5ac86f614a050afb2f24baa35b2
Here's the curlprit 😄, turns out the header is important in patch files, I thought it was not since it was "forced" to apply to the .config file with the patch command.

I didn't use your patch, I tried using the same kernel .config in case the issue was in the one corresponding to my kernel.
Turns out my patch file was the issue all along.

Thanks for your help !
